cmake/PACE_Docs.cmake

```
@@ -0,0 +1,65 @@
+# Build documentation
+
+set(DOCS_DIR "${Horace_ROOT}/documentation/user_docs" CACHE STRING "Directory containing user documentation")
+
+find_program(sphinx-build NAMES sphinx-build)
+find_program(pdflatex NAMES pdflatex)
+find_program(latexmk NAMES latexmk)
+if (sphinx-build)
+  if (UNIX)
+    add_custom_target(denullify-docs
+      COMMAND sed -i -r "/\[NULL\]/d" "${DOCS_DIR}/build/html/*html"
+      DEPENDS build-docs
+      )
+  else()
+    add_custom_target(denullify-docs
+      COMMAND powershell -ExecutionPolicy Bypass -command "Foreach($f in Get-ChildItem -Path '${DOCS_DIR}/build/html' -Filter *.html) {(Get-Content $f.FullName) | Where-Object {$_ -notmatch '\\[NULL\\]'} | Set-Content $F.FullName}"
```

Splitting this command over a few lines would make it more readable:
```suggestion
      COMMAND
        powershell
          -ExecutionPolicy Bypass
          -command "\
            Foreach($f in Get-ChildItem -Path '${DOCS_DIR}/build/html' -Filter *.html) { \
              (Get-Content $f.FullName) \
                | Where-Object {$_ -notmatch '\\[NULL\\]'} \
                | Set-Content $F.FullName \
            }"
```